mitiq/cdr/cdr_execution.py

```diff
@@ -13,26 +13,25 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-""" Functions for execution of CDR mitigation on circuit of interest"""
+"""API for using Clifford Data Regression (CDR) error mitigation."""
 
-import numpy as np
+from typing import List, Union, Callable, Sequence
 
+import numpy as np
 from scipy.optimize import curve_fit
 
 from cirq.circuits import Circuit
 
-from typing import List, Union, Callable, Optional
-
 from mitiq.cdr.clifford_training_data import generate_training_circuits
-
 from mitiq.cdr.data_regression import (
-    scale_noise_in_circuits,
     construct_training_data_floats,
     construct_circuit_data_floats,
     linear_fit_function,
 )
+from mitiq.zne.scaling import fold_gates_at_random
 
 
+# TODO: Allow for any QPROGRAM, not just a cirq.Circuit.
```

Comment:
Only relevant changes are in this function. The rest is just merging mitiq/master.